src/middlewares/cors.ts

```diff
@@ -18,7 +18,7 @@ export const corsHandler = cors(async (req: any, callback: any) => {
     if (!origin || allowedOrigins.indexOf(origin) > -1) {
         callback(null, {
             credentials: true,
-            origin: allowedOrigins,
+            origin: '*',
```

Comment:
@GarfDev We probably need to do a similar trick here as in the cors module of the API since I assume you changed this for the purpose of the SDK being able to use the signin flow. I will create a task for that.